jdaviz/configs/imviz/plugins/footprints/footprints.py

```diff
@@ -153,6 +156,27 @@ def marks(self):
                           if hasattr(viewer, 'figure')}
                 for overlay in self.overlay.choices}
 
+    @staticmethod
+    def _file_parser(path):
+        def _ensure_sky(region):
+            if isinstance(region, regions.Regions):
+                return np.all([_ensure_sky(reg) for reg in region.regions])

```

Comment:
I think this is too strict because technically you can skip over the invalid ones and still load the valid ones. But then again, we are going to add pixel region support soon, not I guess we can just leave this be for now.

Reply:
Yes, but then it can also result in confusing behavior if the user misses a warning somewhere.  But I agree, we may eventually want to loosen up any error checking to warning messages once we decide exactly which cases are covered and which are not.